lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -101,6 +101,18 @@ def preview?
             mode == Renderer::PREVIEW_MODE
           end
 
+          def rand_hex(n)
+            SecureRandom.hex(n)
+          end
+
+          def rand_name
+            NameGenerator.new.next_random_name
```

Comment:
Please use `generate_next_random_name` instead, this one depends on setting but we want the macro to work in any setting.